src/spirc.rs

```diff
@@ -170,6 +172,16 @@ impl<D: SpircDelegate> SpircManager<D> {
             protocol::spirc::MessageType::kMessageTypePause => {
                 self.delegate.pause();
             }
+            protocol::spirc::MessageType::kMessageTypeNext => {
+            	self.index = (self.index + 1) % self.tracks.len() as u32;
+                let track = self.tracks[self.index as usize];
```

Comment:
So this would mean it loops through approx. 50 songs depending on first load?

codeman: Could you elaborate on the Replace type when exactly is it called? I can produce it by adding something to my queue, but any other triggers? Thanks